pkg/plugins/platform/local.go

```diff
@@ -201,9 +201,23 @@ func (l *LocalApp) ImportDB(imPath string, extPath string) error {
 		}
 	}
 
-	err = l.Exec("db", true, "./import.sh")
+	matches, err := filepath.Glob(filepath.Join(dbPath, "*.sql"))
 	if err != nil {
-		return fmt.Errorf("failed to execute import: %v", err)
+		return err
+	}
+
+	if len(matches) < 1 {
+		err = fileutil.PurgeDirectory(dbPath)
+		if err != nil {
+			return fmt.Errorf("no .sql files found to import, failed to cleanup .ddev/data: %v", err)
+		}
+		return fmt.Errorf("no .sql files found to import")
+	}
+
+	fmt.Println("Importing database...")
+	err = l.Exec("db", true, "bash", "-c", "cat /db/*.sql | mysql")
```

Comment:
Thank you :) Not to be ungrateful, I'd love to see us actually connect to the database and feed it the query someday, rather than exec-ing a script over there.